app_dart/test/src/service/fake_gerrit_service.dart

```diff
@@ -24,7 +24,7 @@ class FakeGerritService extends GerritService {
         );
 
   List<String> branchesValue;
-  static const List<String> _defaultBranches = <String>['main'];
+  static const List<String> _defaultBranches = <String>['refs/heads/master'];
```

Comment:
I know this is only in the Fake service, but was this suppose to be a git ref?

Reply:
Yes, this would always hit the "not found" scenario in a test (if we had checked the logs)

See https://flutter-review.googlesource.com/projects/recipes/branches for example.